docs/user-guide/certificates-configuration-questionnaire.md

```diff
@@ -0,0 +1,118 @@
+# Zowe certificates questionnaire
+
+Answering [the questions](#certificates-configuration-questionnaire) in this questionnaire will help to make your choice of certificates setup according to your planned Zowe deployment.
+
+If you need to learn more about the certificates based security terminology, review the [certificates concepts](../appendix/zowe-security-glossary#certificate-concepts) in our Security Glossary.   
+
+If you want to lean more about how Zowe uses digital certificates, visit the [Zowe Certificates Overview](../getting-started/zowe-certificates-overview).
+
+Before you answer [the questions](#certificates-configuration-questionnaire) you could also review the [Certificate Configuration Scenarios](certificate-configuration-scenarios.md) to understand better the certificates configuration options. 
+
+## Certificates configuration options
+
+1. Certificates storage type.
+
+You can instruct Zowe installation to store your certificates in a:
+- File-based keystore/truststore pair
+- IBM keyring
+
+2. Certificate file format.
+
+You need to acquire or generate certificates in the format accepted by the selected storage type.
+Vice-versa, if you already have digital certificates for your servers, you'd need to select the appropriate storage type that can host them. 
+The digital certificate types that can be used by Zowe are: 
+- PKCS12
+- JKS / JCEKS
+
+3. Certificate signing
+
+Every digital certificate must be signed by a public/private Certificate Authority(CA), or they can be self-signed. Recursively, the CA's signing certificates also must be signed, 
+effectively building a trust chain, rooted by a trusted Root Certificate Authority (who's certificate is self-signed). The Root CA can be a public or a private one.     
+
+When the certificates are signed by a public CA, their genuineness can be easily verified due to the transitive trust policy based on the signing certificates chain and the public availability of that certificates.   
+When the CA is a private to the organization, the clients must explicitly request the CA certificate of the private Root CA and then import it to their trust stores in order to apply the transitive trust policy.   
```

Comment:
I think you're missing a word here: "When the CA is a private to the organization ... ." The CA is a private _what_?

Reply:
I meant, "When the CA is private to the organization" i.e. it belongs to the organization, those its certificate is self-signed, and not signed by a public ROOT authority.